core/test/junit/saros/account/XMPPAccountStoreTest.java

```diff
@@ -89,26 +89,16 @@ private void writeAccountFile(File accountFile, String key, String content) thro
   }
 
   private String createAccountFileContent(
-      XMPPAccount activeAccount, List<XMPPAccount> otherAccounts) {
+      XMPPAccount activeAccount, ArrayList<XMPPAccount> configuredAccounts) {
```

Comment:
I do not get idea behind this test. Do you not trust XStream ? What is wrong with creating an XMPPAccountStore, add some accounts and save it. Afterwards you create another XMPPAccountStore and use the existing file. I guess such a test already exists.

Reply:
The idea was to inform a developer if he/she changes the XML format/content of the file and therefore invalidates our old files.

Reply:
You can also use Aliases to allow variable renaming in the future. And I also think adding a comment to the affected classes (e.g if you change XYZ you will broke ABC) is much more helpful than only have one test case that is failing.

Reply:
I will add these additional information to the corresponding lines.

Reply:
Done